cdk/bin/cdk.ts

```diff
@@ -1,7 +1,10 @@
 import 'source-map-support/register';
 import { App } from 'aws-cdk-lib';
 import { AdminConsole } from '../lib/admin-console';
+import { AdminConsoleDynamo } from '../lib/admin-console-dynamo';
 
 const app = new App();
 new AdminConsole(app, 'AdminConsole-CODE', { stack: 'support', stage: 'CODE' });
 new AdminConsole(app, 'AdminConsole-PROD', { stack: 'support', stage: 'PROD' });
+new AdminConsoleDynamo(app, 'AdminConsoleDynamo-CODE', { stack: 'support', stage: 'CODE' });
+new AdminConsoleDynamo(app, 'AdminConsoleDynamo-PROD', { stack: 'support', stage: 'PROD' });
```

Comment:
This adds the new stack, separate from the existing one that contains the loadbalancer, autoscaling group etc

Reply:
so the reason to have this as a separate stack is to decouple the updates between the two, given the dynamo stack will almost never change?

Reply:
It's more my instinct to keep the persistence separate from the app...but I'm not sure it's necessary. The `removalPolicy: RemovalPolicy.RETAIN` part should ensure the tables are never deleted by an accidental cloudformation update.
I'm going to go away and get opinions on the devex channel before going ahead with this!

Reply:
I ended up doing it all in one stack